src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java

```diff
@@ -51,17 +51,19 @@ public CaseDocument bundle(
                 bundleFilename,
                 caseData
             );
-
+        log.info("calling stitching api end point for ", caseData.getLegacyCaseReference());
```

Comment:
@indu2019 this log doesnt work .. it should be 
        log.info("Received ActorId: {}", actorId); like this ..Did you check it locally

Reply:
For more info : read here https://www.slf4j.org/faq.html#logging_performance